lib/fs/index.js

```diff
@@ -117,3 +116,25 @@ if (typeof fs.writev === 'function') {
     })
   }
 }
+
+// fs.realpath.native only available in Node v9.2+
+if (typeof fs.realpath.native === 'function') {
+  exports.realpath.native = u(fs.realpath.native)
+} else {
+  // fallback to a similar method signature that is present before Node v9.2
+  exports.realpath.native = u(fs.realpath)
+
+  // emit warning that fs.realpath.native is not a function (or undefined)
+  const nodeVer = process.version.match(/(\d+)\.(\d+)\.(\d+)/)
```

Comment:
I agree with @RyanZim totally. I think a warning would suggest to only write something like following (as we already have a NodeJS check on the engine level): 

```js
process.emitWarning(
 `fs-extra: fs.realpath.native is not a function. Ensure that fs-extra is imported before other dependencies that can change fs;\nsee: https://github.com/jprichardson/node-fs-extra/pull/953`,
  'Warning',
  'fs-extra-WARN003'
)

Reply:
Your wish is my command. 🤗